packages/components/src/Datalist/Datalist.scss

```diff
@@ -24,6 +24,12 @@ $tc-datalist-item-height: 4rem !default;
 	}
 }
 
+.tc-datalist {
+	:global(.icon-caret) {
```

Comment:
if `.icon-caret` is set by typeahead can we move this fix to typeahead? I mean typeahead can be affected too?